src/styles/blocks/product.scss

```diff
@@ -0,0 +1,155 @@
+/* Products */
+
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: center;
+  justify-items: center;
+
+  grid-template-rows: 2fr;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: $horizontal-card-gap $vertical-card-gap;
```

Comment:
The use of CSS variables alongside SCSS variables can be confusing. It's generally best to stick with one or the other. In this case, since you're using SCSS, I would recommend using SCSS variables for column count as well. This would make your code more consistent and easier to maintain.